using-mobile-extensions/adobe-analytics-edge/adobe-analytics-edge-release-notes.md

```diff
@@ -0,0 +1,17 @@
+# Release Notes
+
+## January XX, 2021 (TO BE UPDATED)
```

Comment:
In my opinion we can either keep this page empty until the 1.0.0 release or add a note about beta for now.